### PR TITLE
Add notes about Zoom everywhere

### PR DESCRIPTION
**Pull request by gxalpha**
This should reduce the number of issues regarding Zoom and virtual cameras.
Please comment if you think there are more places where this may be needed.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ assignees: ''
 
 ---
 
+_Attention: While Zoom says they re-enabled virtual cameras, only certain ones they whitelisted work. Currently, this plugin does not work with Zoom!_
```
Comment (johnboiles) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Idea: You might want to link to the codesigning workaround in the wiki

File: README.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ If, after you donate to the OBS Project, you also want to send some cash my way
...
+* Zoom, Slack, Webex, Skype and probably some other applications have disabled virtual cameras by default via [application restrictions](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_cs_disable-library-validation?language=objc). Check out [the wiki](https://github.com/johnboiles/obs-mac-virtualcam/wiki/Compatibility) to see if your app is supported. Please [edit the wiki](https://github.com/johnboiles/obs-mac-virtualcam/wiki/Compatibility/_edit) if you try other software that we should include in that list. In most cases you can work around these restrictions by [re-codesigning those applications](https://github.com/johnboiles/obs-mac-virtualcam/wiki/Compatibility#apps-dont-allow-dal-plugins). *Attention: While Zoom says they re-enabled virtual cameras, only certain ones they whitelisted work. Currently, this plugin* ***does not work with Zoom!***
```
Comment (johnboiles) [inline README.md:21]: At very least add a link to the compatibility page here to reference the workaround.

**Comment by johnboiles** [comment]
Thanks @gxalpha !

**Comment by gxalpha** [comment]
This could be reverted now, right?

**Pull request merged**